src/main/scala/lotto/domain/WinningLotto.scala

```diff
@@ -0,0 +1,26 @@
+package lotto.domain
+
+import scala.util.Random
+
+class WinningLotto (val lotto: Lotto, val bonusNumber: LottoNumber) {
+    require(!lotto.numbers.contains(bonusNumber), "보너스 숫자는 로또의 숫자와 겹칠 수 없습니다.")
+
+    def this(lotto: List[Int], bonusNumber: Int) = this(Lotto(lotto), LottoNumber(bonusNumber))
+    override def toString: String = s"LottoNumber : $lotto \t bonusNumber : $bonusNumber"
+
+}
+
+object WinningLotto {
+
+    def apply(lotto: Iterable[Int], bonusNumber: Int): WinningLotto = {
+        new WinningLotto(lotto.toList, bonusNumber)
```

Comment:
따로 toList하지 않고 사용하는 방법은 어떤가요? Lotto.[apply](https://github.com/ekzm8523/scala-lotto/pull/1/files#diff-e73ba59f5ec2e0ef86b1110af8ab462a39ccd77a5c8816a19bb883c4bdbceff3R14) 를 바로 이용하면 될 것 같아요
```suggestion
        new WinningLotto(Lotto(numbers), bonusNumber)
```